server/server.go

```diff
@@ -200,7 +204,10 @@ func (server *HTTPServer) readReportMetainfoForCluster(writer http.ResponseWrite
 		return
 	}
 
-	reports, lastChecked, storedAt, err := server.Storage.ReadReportForCluster(orgID, clusterName)
+	reports, lastChecked, storedAt, _, err := server.Storage.ReadReportForCluster(
```

Comment:
Why not using `gatheredAt` here?

Reply:
I'm not implementing yet the update for this endpoint, it will be in a separated PR because it is common for both V1 and V2 endpoints in Smart Proxy. I need to split the logic in Smart Proxy first for that endpoint and then, I will be able to return the gathered_at field here too :-)

Reply:
I'm not implementing yet the update for this endpoint, it will be in a separated PR because it is common for both V1 and V2 endpoints in Smart Proxy. I need to split the logic in Smart Proxy first for that endpoint and then, I will be able to return the gathered_at field here too :-)